### ISSUE TITLE
WinterCG

### ISSUE THREAD

**Issue by CanadaHonk** [Igalia/webengineshackfest#29]
### Logistics

* Date & Time: [Wednesday 5th June at 10:00 CEST (local time)](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20240605T0800)
* URL: https://meet.jit.si/WEH2024-WinterCG
* Notes: https://github.com/Igalia/webengineshackfest/wiki/2024-WinterCG

### Facilitator(s)

@CanadaHonk (Mozilla), @andreubotella (Igalia), @littledan (Bloomberg), @lucacasonato (Deno)

### Summary

[WinterCG](https://wintercg.org) explanation and general discussion (agenda todo).

### Type

Onsite

**Comment by CanadaHonk** [comment]
For scheduling, this could go right after #27 (or be merged into a longer single session/etc)

**Comment by CanadaHonk** [comment]
Slides for WinterCG https://docs.google.com/presentation/d/1HPT3wEofRRyORD77k3IfPONkNOKtUDRQ0EpS9r3f1fE/edit?usp=sharing

**Comment by littledan** [comment]
[Ecma-doc-TC55_Scope_Edited_sh.docx](https://github.com/user-attachments/files/15582438/Ecma-doc-TC55_Scope_Edited_sh.docx)
[Ecma-doc-TC55_WorkingModel_Edit_sh.docx](https://github.com/user-attachments/files/15582440/Ecma-doc-TC55_WorkingModel_Edit_sh.docx)

**Comment by mrego** [comment]
Notes have been moved to the wiki: https://github.com/Igalia/webengineshackfest/wiki/2024-WinterCG
